vaadin-icons-flow-demo/src/main/java/com/vaadin/flow/component/icon/demo/allIcons/AllIcons.java

```diff
@@ -0,0 +1,12 @@
+package com.vaadin.flow.component.icon.demo.allIcons;
```

Comment:
`package com.vaadin.flow.component.icon.demo.allIcons;` rename package to be all lowercase